python/docs/introduction.rst

```diff
@@ -64,10 +64,12 @@ Key Concepts
         the name used in both Feldera Program and Pipeline.
       - The second parameter here is :class:`.FelderaClient` that we created above.
 
-* :meth:`.SQLContext.run_to_completion`
-   - Runs this Feldera pipeline to completion. Normally this means until the EoF
+* :meth:`.SQLContext.wait_for_completion`
+   - Blocks this Feldera pipeline until completion. Normally this means until the EoF
```

Comment:
It's "EOF" (with an uppercase O). When writing, and using an abbreviation for the first time, first give word followed by abbreviation, e.g., "end-of-file (EOF)".